taccsite_cms/static/site_cms/css/src/_imports/objects/o-section.css

```diff
@@ -39,6 +39,9 @@ Styleguide Objects.Section
 /* Block */
 
 .o-section {
+  --border-width: var(--global-border-width--normal);
+  --border-width--neg: calc(-1 * var(--border-width));
```

Comment:
**Notice.** Just a couple variables for short-term convenience. _FP-1470 (later) will bring more._

Reply:
Variables were removed. I hide my comment.